pkg/plugins/resources/postgres/plugin.go

```diff
@@ -34,7 +35,7 @@ func (p *plugin) NewResourceStore(pc core_plugins.PluginContext, config core_plu
 	}
 	switch cfg.DriverName {
 	case postgres.DriverNamePgx:
-		return NewPgxStore(pc.Metrics(), *cfg)
+		return NewPgxStore(pc.Metrics(), *cfg, multitenant.DefaultPgxConfigCustomization{})
```

Comment:
Should we be wiring in the ConfigCustomization here?

Reply:
It almost feels for this bit that runtimeContext config is a little strange but I guess it's ok for now

Reply:
yep, this should be `pc.ConfigCustomization()`